src/awake.h

```diff
@@ -1122,6 +1123,7 @@ enum {
 #define ITEM_WEAR_FACE   23
 #define ITEM_WEAR_THIGH  24
 #define ITEM_WEAR_MAX    25
```

Comment:
X_MAX is used by the code to indicate 'the acceptable values for this are 0 up to this number', so having the mouth flag after wear_max means it won't be accounted for in a lot of the code. Swap these so mouth is 25 and max is 26.